devnet-sdk/system/chain.go

```diff
@@ -61,22 +62,26 @@ type chain struct {
 	id     string
 	rpcUrl string
 
-	users    map[string]Wallet
-	clients  *clientManager
-	registry interfaces.ContractsRegistry
-	mu       sync.Mutex
+	users       map[string]Wallet
+	clients     *clientManager
+	registry    interfaces.ContractsRegistry
+	mu          sync.Mutex
+	chainConfig *params.ChainConfig
```

Comment:
Feels like we need a unification here. https://github.com/ethereum-optimism/optimism/pull/14668 adds `descriptors.ChainConfig`.